.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,22 @@
+name: Continuous Integration
+on:
+  push:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    name: Build and Run Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 #v3.5.3
+
+      - name: Create dummy .env
+        run: touch .env
```

Comment:
Right thing to do, but it's really annoying that docker-compose insists on this!